packages/interactivity/src/store.js

```diff
@@ -38,6 +38,49 @@ const getSerializedState = () => {
 const rawState = getSerializedState();
 export const rawStore = { state: deepSignal( rawState ) };
 
+/**
+ * Extends the Interactivity API global store with the passed properties.
+ *
+ * These props typically consist of `state`, which is reactive, and other
+ * properties like `selectors`, `actions`, `effects`, etc. which can store
+ * callbacks and derived state. These props can then be referenced by any
+ * directive to make the HTML interactive.
+ *
+ * @example
+ * ```js
+ *  store({
+ *    state: {
+ *      counter: { value: 0 },
+ *    },
+ *    actions: {
+ *      counter: {
+ *        increment: ({ state }) => {
+ *          state.counter.value += 1;
+ *        },
+ *      },
+ *    },
+ *  });
+ * ```
+ *
+ * The code from the example above allows blocks to subscribe and interact with
+ * the store by using directives in the HTML, e.g.:
+ *
+ * ```html
+ * <div data-wp-interactive>
+ *   <button
+ *     data-wp-text="state.counter.value"
+ *     data-wp-on--click="actions.counter.increment"
+ *   >
+ *     0
+ *   </button>
+ * </div>
+ * ```
+ *
+ * @param {Object} properties         Properties to be added to the global store.
+ * @param {Object} [properties.state] State to be added to the global store. All
+ *                                    the properties included here become reactive.
+ * @param {Object} [properties.block] Other properties to be added to the global store.
```

Comment:
I believe this is not the correct way of documenting a variable that uses the spread operator. It seems to work somehow, but only for the docs, not for types. We could maybe remove it.

Reply:
Up to you 🙂

Reply:
I'll remove it for now, then.